src/com/rockbite/bootcamp/IShop.java

```diff
@@ -0,0 +1,32 @@
+package com.rockbite.bootcamp;
+
+import java.io.IOException;
+import java.util.List;
+
+/**
+ * Shop API
+ */
+public interface IShop {
+
+    /**
+     * @return list of products available in the shop
+     */
+    List<Product> showProducts();
```

Comment:
showProducts suggests that it will somehow be displaying the products. A better name would be getAvailableProducts,